.pylintrc

```diff
@@ -279,7 +279,7 @@ init-import=no
 
 # List of qualified module names which can have objects that can redefine
 # builtins.
-redefining-builtins-modules=six.moves,future.builtins
+redefining-builtins-modules=future.builtins
```

Comment:
Can't the `future.builtins` also not be removed? Or does that have nothing to do with the `__future__` statements?

Reply:
Hh, forget to remove it, cause I did not take care of __future__ at previous PR.